static/app/components/duration/duration.tsx

```diff
@@ -0,0 +1,45 @@
+import type {HTMLAttributes} from 'react';
```

Comment:
shouldn't we name this file `index.tsx`?

Reply:
Can't do that now, it'll collide with the existing `components/duration.tsx` file which i plan to replace in the near future.